### DISCUSSION TITLE
每隔几分钟会重新连接并登录

### DISCUSSION THREAD

**Discussion by CrazyGreyWolf** [mamoe/mirai#1048] [Q&A]
![image](https://user-images.githubusercontent.com/47654285/108811121-9ef9d380-75e7-11eb-8d31-b911fbfbf091.png)
上图是mcl运行的控制台日志，显示会重新连接并登录

![image](https://user-images.githubusercontent.com/47654285/108811218-d49ebc80-75e7-11eb-8e5a-bce213aca516.png)
这个是bots下的日志，显示重新连接并登录

从bots的日志看到，发现每次打印出 “Send: Heartbeat.Alive” 日志，并且没有响应（正常的是在该日志下很快有另一个日志"Recv: Heartbeat.Alive.Response"表示收到心跳响应包）包，连续两次后，就会重新连接。

我简单看了下源码，是不是PlatformSocket对象的close在哪个地方被调用导致socket被关闭了

这个问题会影响到机器人接收消息，因为如果是上述情况，有时候会导致在重连之前，机器人不能及时收到消息，只能等待下次重连后才能收的到

**Comment by Him188** [comment]
这是一个已知问题

**Reply by CrazyGreyWolf** [reply]
好的。了解了
